Makefile

```diff
@@ -381,17 +381,18 @@ helm-index:
 		result=$$?;\
 		if [ $${result} -eq 0 ]; then\
 			echo "Downloaded $${dl_url}";\
-			mv ./$(CHART_REPO)-$${tag}.tgz .cr-release-packages/;\
+			mkdir -p .cr-release-packages/$${tag}
```

Comment:
@janorn I tested this out locally, it throws a syntax error because ln 384 is missing a `;\` at the end of it.  Otherwise, this change works.  Could you update the PR, then I can merge?